Content.Server/Chemistry/Components/MeleeChemicalPickerComponent.cs

```diff
@@ -0,0 +1,25 @@
+using Content.Shared.FixedPoint;
+
+namespace Content.Server.Chemistry.Components
+{
+    [RegisterComponent]
```

Comment:
documentation and Access attribute

Reply:
similar MeleeInjector component does not have a defined tolerance. I suspect because there is no system with the same name?